src/main/java/core/basesyntax/service/UserDataInvalidException.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+public class UserDataInvalidException extends RuntimeException {
+    private String message;
+
+    public UserDataInvalidException(String message) {
+        this.message = message;
```

Comment:
Use 'super(message);' to pass the message to the superclass constructor, allowing the RuntimeException to handle the message.